### DISCUSSION TITLE
Syntax highliting not working

### DISCUSSION THREAD

**Discussion by TuTAH1** [atom/atom#22938] [Q&A]
I'm new to atom, just installed it and XML syntax highlighting not works, the text just white, while in the right down corner of the editor there's "CRLF    UTF-8    **XML**", so xml is detected, but why it's not highlited?
As I searched on the internet, the xml highlight should be built in Atom and no plugins are required. The file size is 28 MB

**Comment by icecream17** [comment]
Hmm, #22129 doesn't seem to apply to atom/language-xml....

Can you try screenshotting the console? `ctrl+shift+i`

**Reply by TuTAH1** [reply]
![Скриншот 05-09-2021 184236](https://user-images.githubusercontent.com/15982179/132129176-4a30af96-a8e9-4c41-a86e-2f95cbd1bac4.jpg)
